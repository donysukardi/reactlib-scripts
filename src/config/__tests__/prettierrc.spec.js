import prettierRc from '../prettierrc'

test('It matches snapshot', () => {
  expect(prettierRc).toMatchSnapshot()
})
