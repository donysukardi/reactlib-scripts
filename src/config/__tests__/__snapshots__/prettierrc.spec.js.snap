// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches snapshot 1`] = `
Object {
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "printWidth": 80,
  "semi": false,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "useTabs": false,
}
`;
